resources/public/dashboard/index.html

```diff
@@ -155,7 +155,7 @@
   var height = getParameter("height") || 60;
 
   counterAndSparkline()
-  .url("../v4/metrics/mbean/java.lang:type=Memory")
+  .url("../metrics/v1/mbean/java.lang:type=Memory")
```

Comment:
without even testing it, I'm pretty sure all these urls need to change to 'mbeans'.